lighthouse-core/gather/driver.js

```diff
@@ -433,12 +433,14 @@ class Driver {
    *    See https://developer.mozilla.org/en-US/docs/Web/API/Permissions/query.
    */
   queryPermissionState(name) {
+    if (['geolocation', 'notifications', 'midi', 'push'].indexOf(name) === -1) {
```

Comment:
yeah @ebidel this was to remove the `JSON.stringify` in response to @brendankenny's feedback since a malformed name wouldn't ever reach the friendly message. i'd lean toward my original stringify and let Chrome handle the enum check but I don't feel terribly strongly